src/io/flutter/sdk/FlutterCommand.java

```diff
@@ -254,7 +254,7 @@ public GeneralCommandLine createGeneralCommandLine(@Nullable Project project) {
     if (workDir != null) {
       line.setWorkDirectory(workDir.getPath());
     }
-    if (!isDoctorCommand()) {
+    if (!isDoctorCommand() && !(sdk instanceof FlutterSdk.BazelSdk)) {
```

Comment:
I don't know why the existing code is conditionally passing in `--no-color` (and only not passing it in for the doctor command). I wonder if we couldn't just remove the condition and always pass in `--no-color`.

Reply:
Doctor command output becomes very hard to read when IntelliJ passes the --no-color flag.

See gpaste/5549831424573440 for a sample.

The Flutter bazel commands do not all accept --no-color.  The only one that does right now is the test command.

I'll leave this section as-is for now, and if we want to tell IJ to not color doctor command output in the future, we can file an issue for it.

Reply:
I filed https://github.com/flutter/flutter/issues/33321 about this.